detections/endpoint/multiple_invalid_users_failing_to_authenticate_from_host_using_kerberos.yml

```diff
@@ -0,0 +1,41 @@
+name: Multiple Invalid Users Failing To Authenticate From Host Using Kerberos
+id: 001266a6-9d5b-11eb-829b-acde48001122
+version: 1
+date: '2021-04-14'
+author: Mauricio Velazco, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: The following analytic identifies one source endpoint failing to authenticate with multiple invalid domain users using the Kerberos protocol. 
+  This behavior could represent an adversary performing a Password Spraying attack against an Active Directory environment using Kerberos to obtain 
+  initial access or elevate privileges. As attackers progress in a breach, mistakes will be made. In certain scenarios, adversaries may execute a password spraying attack using an invalid list of users. 
+  In this logic, the number of unique users is hardcoded to '10' but can/should be customized to fit especific environments.
+  Event 4768 is generated every time the Key Distribution Center issues a Kerberos Ticket Granting Ticket (TGT). Failure code 0x6 means 
+  'client not found in Kerberos database' (the attempted user is not a valid domain user).\
+
+  This detection will trigger on the domain controller against which the offending host performs the password spraying attack.
+
+  The analytic's returned fields allow analysts to investigate the event further by providing fields like source ip and attempted user accounts.
+search: '`wineventlog_security` EventCode=4768 Result_Code=0x6 | stats dc(Account_Name) AS unique_accounts values(Account_Name) as tried_accounts by Client_Address | where unique_accounts > 10 | `multiple_invalid_users_failing_to_authenticate_from_host_using_kerberos_filter` '
```

Comment:
With the new standard deviation approach, if a gateway client commonly fails to authenticate with user, it will not pass the 3 standard deviations.

Furthermore, I think this detection, just like all others, may require a lookup table or allow list for certain hosts that behave this way. For example, vulnerability scanners.